askama_parser/fuzz/fuzz_targets/fuzz_parser.rs

```diff
@@ -0,0 +1,13 @@
+#![no_main]
+use askama_parser::*;
+use libfuzzer_sys::fuzz_target;
+use std::str;
+
+fuzz_target!(|data: &[u8]| {
+    // fuzzed code goes here
+    if data.len() < 500 {
```

Comment:
How did you decide on the 500-byte boundary? I think it probably makes sense...

Reply:
Ah sorry, you mentioned there was a stack overflow. That seems suspicious, maybe I'll have a look at that.

Reply:
I'll see what happens if I increase the limit locally.